src/containers/AsideNavigation/AsideNavigation.tsx

```diff
@@ -115,46 +111,8 @@ interface AsideNavigationProps {
     setSettingValue: (name: string, value: string) => void;
 }
 
-const items: MenuItem[] = [
-    {
-        id: CLUSTER_PAGES.tenants.id,
-        title: 'Databases',
-        icon: databasesIcon,
-        iconSize: 20,
-        location: createHref(routes.cluster, {
-            activeTab: CLUSTER_PAGES.tenants.id,
-        }),
-        locationKeys: ['/tenant'],
-    },
-    {
-        id: CLUSTER_PAGES.nodes.id,
-        title: 'Nodes',
-        icon: databaseIcon,
-        iconSize: 20,
-        location: createHref(routes.cluster, {activeTab: CLUSTER_PAGES.nodes.id}),
-        locationKeys: ['/node'],
-    },
-    {
-        id: CLUSTER_PAGES.storage.id,
-        title: 'Storage',
-        icon: storageIcon,
-        iconSize: 20,
-        location: createHref(routes.cluster, {
-            activeTab: CLUSTER_PAGES.storage.id,
-        }),
-        locationKeys: ['/storage'],
-    },
-    {
-        id: CLUSTER_PAGES.cluster.id,
-        title: 'Cluster',
-        icon: clusterIcon,
-        iconSize: 20,
-        location: createHref(routes.cluster, {
-            activeTab: CLUSTER_PAGES.cluster.id,
-        }),
-        locationKeys: ['/cluster/cluster'],
-    },
-];
+// FIXME: add items or delete
+const items: MenuItem[] = [];
```

Comment:
I left it as it was to keep logic and to fill it with new items on the next iteration